decidim-meetings/app/views/decidim/meetings/admin/registrations/_form.html.erb

```diff
@@ -6,7 +6,10 @@
         <div class="flex items-center gap-x-4 ml-auto">
           <%= link_to t(".invites"), meeting_registrations_invites_path(meeting), class: "button button__sm button__secondary #{"disabled" unless allowed_to? :read_invites, :meeting, meeting: meeting}" %>
           <% if allowed_to? :export_registrations, :meeting, meeting: meeting %>
-            <span class="exports dropdown tiny button button__sm button__secondary button--simple button--title" data-toggle="export-dropdown"><%= t "actions.export", scope: "decidim.admin" %></span>
+            <span class="exports dropdown tiny button button__sm button__secondary button--simple button--title" data-toggle="export-dropdown">
```

Comment:
I think i have fixed the color on #11696
![image](https://github.com/decidim/decidim/assets/105683/c8d397c8-763c-4085-ae20-cd6c9a779dac)

Reply:
Yeah, I have to admit that I missed that one 😄 
If its already fixed then we can move forward that one with the fix?